.yarnrc.yml

```diff
@@ -9,5 +9,6 @@ plugins:
 unsafeHttpWhitelist:
   - localhost
 
-# This path should always match the one in .yarnrc
-yarnPath: .yarn/releases/yarn-2.0.0-rc.33.js
+yarnPath: .yarn/releases/yarn-2.0.0-rc.36.js
+
+httpTimeout: 120000
```

Comment:
`httpTimeout` is not available in Yarn 2.0.0-rc.33, so I update to the latest version.